developer-docs-site/docs/tutorials/build-e2e-dapp/4-fetch-data-from-chain.md

```diff
@@ -57,7 +57,7 @@ function App (
 The `account` object is `null` if there is no account connected; when an account is connected, the `account` object holds the account information, including the account address.
 
 3. Next, we want to fetch the account’s TodoList resource.
-   Begin by importing `useEffect` by using `jsx import useEffect from "react"; `
+   Begin by importing `useEffect` by using `import { useEffect } from "react"; `
```

Comment:
nit

might be useful to have it as
```
```jsx
import { useEffect } from "react";
```
```
for easy copy-past